### DISCUSSION TITLE
contributions to the mindsdb website

### DISCUSSION THREAD

**Discussion by goswamianshuman** [mindsdb/mindsdb#8561] [Get help] [answered]
Hello, I have found a frontend bug on mindsdb.com, can anyone help me by telling me the location for the website's frontend repository? I'm excited to contribute to it.

thank you in advance!

**Comment by ZoranPandovski** [comment]
Hey @goswamianshuman the website code is not open source so please feel free to open a bug

**Reply by goswamianshuman** [reply]
Hey @ZoranPandovski , so should I just create a new issue marking as a bug and add all the UI fixes there? 😄

**Reply by ZoranPandovski** [reply, marked as answer]
No, you can't submit the UI fixes thorough the issue :) Just report the bug. Thanks
